README.md

```diff
@@ -1,3 +1,5 @@
+**Please consider moving to [appium-mac2-driver](https://github.com/appium/appium-mac2-driver) for newer macOS versions**
```

Comment:
I would rather explicitly mention which versions (e.g. macOS 10.15 Catalina or newer)

Reply:
Also, it makes sense to mention that the Appium team is not going to continue the further development of this driver